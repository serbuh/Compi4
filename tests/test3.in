// Simple assignments and if statements

K x = 300 K;
C y = 100 C;

if(true) {
  print "Pass 1";
}

if(false) {
  print "This shouldn't be printed #1";
}

if(x<y) {
  print "Pass 2";
}

if(5<12) {
  print "Pass 3";
}

if(4>4) {
  print "This shouldn't be printed #2";
}

if (300 K < 500 K) {
  print "Pass 4";
}

int i = 8;

if(i == 8) {
  print "Pass 5";
}
