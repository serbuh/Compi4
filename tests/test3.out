Pass 1
Pass 2
Pass 3
Pass 4
Pass 5
